### ISSUE TITLE
[Feature request] Intermediate room list style

### ISSUE THREAD

**Issue by KaKi87** [SchildiChat/SchildiChat-android#182]
**Is your feature request related to a problem? Please describe.**
The default and non-customizable room list style on Android is `Roomy: big avatar with two-line preview`.

**Describe the solution you'd like**
The `Intermediate: medium sized avatar with single-line preview` room list style would also be offered.

**Describe alternatives you've considered**
None

**Additional context**

| `Roomy` room list style (actual screenshot) | `Intermediate` room list style (forged preview) |
| --- | ----------- |
| ![](https://user-images.githubusercontent.com/21284089/221808373-babb3ea4-504e-4c0f-9c64-52ceb7fd1d7b.jpeg) | ![](https://user-images.githubusercontent.com/21284089/221808376-0357f351-0017-4245-b7fb-015fc97aa598.png) |
| ![](https://user-images.githubusercontent.com/21284089/221808364-cea03f39-4c74-42c5-bd5b-254841818065.jpeg) | ![](https://user-images.githubusercontent.com/21284089/221808378-0eb72490-9a80-4338-8f23-265d122e3be9.png) |

---

Thanks

**Comment by bitokulo** [comment]
A must have!

**Comment by ChaosNicro** [comment]
Agreed, Schildi is a much smoother experience compared to Fluffy, but it looks worse. The roomy style feels very blocky and harsh.
